src/renderer/mod.rs

```diff
@@ -307,9 +313,22 @@ impl Renderer {
         };
 
         let settings = self.settings.get::<RendererSettings>();
+        let size = root_canvas.base_layer_size();
+        let screen_rect = Rect::from_xywh(0.0, 0.0, size.width as f32, size.height as f32);
         let root_window_regions = root_windows
             .into_iter()
-            .map(|window| window.draw(root_canvas, default_background, grid_scale))
+            .map(|window| {
+                window.draw(
+                    root_canvas,
+                    (opacity * 255.0) as u8,
+                    Background {
+                        color: default_background.with_a((255.0 * background_transparency) as u8),
+                        image: background_image,
+                    },
+                    &screen_rect,
+                    grid_scale,
+                )
+            })
```

Comment:
The rendering stops working after resizing the window with `neovide_opacity` set and maximizing the window
![image](https://github.com/user-attachments/assets/5f9d9292-8345-4e2d-aed4-3bb1065f0858)